src/input/JoystickInputDriver.cc

```diff
@@ -93,3 +95,8 @@ const std::string & JoystickInputDriver::get_name() const
     static std::string name = "JoystickInputDriver";
     return name;
 }
+
+void JoystickInputDriver::setJoystickNr(std::string jnr)
+{
+    nr=jnr;
```

Comment:
Prefix member access with `this->` to make it more clear what's being changed.
Also (Out of scope for this PR), more descriptive variable names would make code more readable by future readers.

Reply:
changed - thank you for your suggestion.